ui/components/institutional/signature-mismatch-banner/signature-mismatch-banner.test.tsx

```diff
@@ -60,4 +68,14 @@ describe('MMISignatureMismatchBanner', () => {
 
     expect(getByText(mismatchAccountText)).toBeInTheDocument();
   });
+
+  it('should not display for Sign-in with Ethereum signatures', () => {
+    const { container } = render({
+      currentConfirmationProps: {
+        ...signatureRequestSIWE,
+      },
+    });
+
+    expect(container.querySelector('.mm-banner-alert')).not.toBeInTheDocument();
```

Comment:
Minor, but for consistency should we use the same selector as above in the positive test? Or change both to classes?

Reply:
hi @matthewwalsh0! Thanks for the suggestion

using getByText and not retrieving a result would result in an error
> TestingLibraryElementError: Unable to find an element with the text:

I do like the test above since it checks the content. I added the class check to the test above to display the inverse checks more explicitly